### PR TITLE
feat: Get Redis keys count lego

### PR DESCRIPTION
**Pull request by enerj**
Implements https://github.com/unskript/Awesome-CloudOps-Automation/issues/84

## Description
Created lego for getting redis keys count by given pattern.

### Testing
Tested lego on [sandbox](https://us.app.unskript.io/)
image

### CODE REVIEW

**Comment by enerj** [comment]
Hello @abhishek-unskript, can you or somebody in the team please check my changes?

**Comment by abhishek-unskript** [comment]
+ Jayasimha.

Thanks Yuriy!

File: Redis/legos/redis_get_keys_count/redis_get_keys_count.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+@beartype
+def redis_get_keys_count_printer(output):
+    if output is None:
+        return
+    pprint.pprint({"Matched keys count": output})
+
+@beartype
+def redis_get_keys_count(handle, pattern: str):
+    """redis_get_keys_count returns the matched keys count.
+
```
Comment (jayasimha-raghavan-unskript) [inline Redis/legos/redis_get_keys_count/redis_get_keys_count.py:24]: Please add :param: pattern to the comment section as we use Sphinx to generate lego documentation.

**Review by jayasimha-raghavan-unskript — CHANGES_REQUESTED** [review]
Awesome Yuriy!

Minor comment regarding documentation. Could you please take care of it. Thank you for your Awesome Contribution!

**Review by jayasimha-raghavan-unskript — APPROVED** [review]

File: Redis/legos/redis_get_keys_count/redis_get_keys_count.py (hunk 1 above)
Comment (enerj) [reply Redis/legos/redis_get_keys_count/redis_get_keys_count.py:24]: My fault, added. Thanks @jayasimha-raghavan-unskript

**Pull request merged**